### ISSUE TITLE
Content of fillable forms lost after save to PDF

### ISSUE THREAD

**Issue by markwmuller** [xournalpp/xournalpp#462]
The content of fillable forms is lost after xournal++ export to PDF.

To replicate:
1. Download (e.g.) foersom.com/net/HowTo/data/OoPdfFormExample.pdf and fill out some forms (I did this using evince). Save to PDF. 
2. Open PDF with Xournal++, and export to PDF.
3. Open resulting PDF in evince, confirm contents of fillable forms is gone.

Expected result: Form contents preserved. 

FWIW, using Xournal allows to save the forms with content preserved. 

Alternative workaround is to print to file from Evince, and open result in Xournal++ (but this is annoying, and easily forgotten).

**Comment by andreasb242** [comment]
Should be fixed.

I use now the standard GLib APi, and export the PDF with Cairo.
The old implementation will still be available with -DADVANCED_PDF_EXPORT_POPPLER=ON
Currently Bookmarks are not exported, but with Cairo 1.16 it will also be possible.
So this should solve a lot of problems I hope.

**I Copy this texts to all related tickets **

**Comment by WavesOfSalt** [comment]
Not fixed at all, stable or nightly. Seems like the single page that was saved was due to it having handwriting, a signature. But I tried it and it didn't work. Then, it suddenly decided to hide my signature as if it were a lower layer than the signature box.

Edit: It appears I found the problem. Something happens with the layers (?) when exporting the PDFs. It appears every single box gets overlayed over text, with unpredictable behavior if the page has both text and handwriting.
